### PR TITLE
Add Homepage

### PR DESCRIPTION
**Pull request by piyush-pawar-17**

### CODE REVIEW

File: src/components/App.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+import { getState, getStore } from '../store';
+
+import Navbar from './Navbar';
+import HomePage from './HomePage';
+
+import '../sass/main.scss';
+
+// Firebase
+firebase.initializeApp({
```
Comment (anshumanv) [inline src/components/App.js:14]: Add in .env?

**Review by anshumanv — APPROVED** [review]

File: src/components/Navbar.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+import { Link } from 'react-router-dom';
+
+const Navbar = () => (
+	<nav className="navbar">
+		<div className="container">
+			<Link to="/">Home</Link>
+			<Link to="/">About</Link>
+			<Link to="/">Team</Link>
+			<Link to="/">Timeline</Link>
+			<Link to="/">Event</Link>
```
Comment (Monal5031) [inline src/components/Navbar.js]: Events?

Comment (piyush-pawar-17) [reply src/components/Navbar.js]: ok

**Review by DivyeshPuri — APPROVED** [review]
GOD Play :godmode:

**Review by Monal5031 — APPROVED** [review]
:100:

**Pull request merged**